c-sharp/DataStructures/DataStructures/LinkedList.cs

```diff
@@ -72,6 +72,24 @@ public void InsertAfter(int value, int newValue)
         }
           current = current.Next;
         }
+    }
+
+    public int FindKthValue(int value)
+    {
+      Node current = Head;
+      current.Next = current;
```

Comment:
This is going to create an infinite list - current, which just got set to `Head`, is now pointing to itself!

Traversal tattoo is:

```csharp
Node current = Head;
while (current != null)
{
    // Do something interesting with current
    current = current.Next;
}
```